product_docs/docs/epas/16/application_programming/ecpgplus_guide/05_building_executing_dynamic_sql_statements.mdx

```diff
@@ -92,7 +92,7 @@ EXEC SQL WHENEVER SQLERROR DO handle_error();
 Then, the example connects to the database using the credentials specified on the command line:
 
 ```sql
-EXEC SQL CONNECT :argv[1];
+EXEC SQL CONNECT edb;
```

Comment:
This change breaks the example and hardwires it to the edb database contrary to line 92.

Reply:
Hi,
Greetings.

The reason I suggest such change is because your first example is defined as a program without any argument.
However, with "CONNECT :argv[1]", there is an argument left in the program.

This might not match your program example's purpose.

Kind Regards,
Yuki Tei